### PR TITLE
Stratis support update for stratis 3.1.0

### PR DESCRIPTION
**Pull request by vojtechtrefny**
This uses the updated tool for predicting pool and filesystem metadata sizes and adds support for specifying size when creating a file system.

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
This should be now ready for review. There still might be some issues, but with #1060 I'm pretty sure at least the basics work as expected. The rawhide failure is caused by Python 3.11, pylint doesn't work with it yet.

File: blivet/devicefactory.py [hunk 1]
```diff
@@ -2095,13 +2103,56 @@ def _get_total_space(self):
 
             return space
```
Comment (japokorn) [inline blivet/devicefactory.py:2104]: Is there a reason that this branch is the only one returning a result?

**Review by japokorn — APPROVED** [review]
Looks good to me.

File: blivet/devicefactory.py (hunk 1 above)
Comment (vojtechtrefny) [reply blivet/devicefactory.py]: Yes, this `return` is unnecessary, the `return` at the end of the function will take care of all branches. Fixed, thanks.

**Pull request merged**